src/Team/Domain/Model/MemberId.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Team\Domain\Model;
+
+enum MemberId: string
+{
+    case ArnaudDeAbreu = 'arnaud-de-abreu';
+    case EnzoSantamaria = 'enzo-santamaria';
+    case FelixEymonot = 'felix-eymonot';
+    case HugoAlliaume = 'hugo-alliaume';
+    case JulesPietri = 'jules-pietri';
+    case JeremyRomey = 'jeremy-romey';
+    case MathiasArlaud = 'mathias-arlaud';
+    case RobinChalas = 'robin-chalas';
+    case ValmontPehautPietri = 'valmont-pehaut-pietri';
```

Comment:
Let's wait on each guy to tell if he's ok to be on this page:
- [ ] ArnaudDeAbreu
- [ ] EnzoSantamaria
- [ ] FelixEymono
- [ ] HugoAlliaume
- [ ] JulesPietri
- [ ] JeremyRomey
- [x] MathiasArlaud
- [x] RobinChalas
- [ ] ValmontPehautPietri

Reply:
We must add Yazid as well :slightly_smiling_face: